File: src/main/java/k/fixu/decode/service/CodeService.java
package k.fixu.decode.service;

import k.fixu.decode.domain.CodeString;
import k.fixu.decode.domain.CodeUser;

/**
 * @author karrb
 * @version 1.0
 * @date 2019/02/19 14:59
 */
public interface CodeService {

    void save(CodeString codeString);

    CodeString findOne(Long id,String pwd);
}

File: src/main/webapp/js/service/codeService.js
//服务层
app.service('codeService', function ($http) {

    //加密
    this.code = function (entity) {
        return $http.post('code/code.do', entity);
    };

    //解密
    this.decode = function (entity) {
        return $http.post('code/decode.do', entity);
    }

});
